src/components/hero-decorative/hero-decorative.js

```diff
@@ -1,29 +1,29 @@
 import './hero-decorative.scss'
 import template from './hero-decorative.pug'
-import heroData from '../../data/data.json'
+import heroDecorativeData from '../../data/data.json'
```

Comment:
I think the name of this variable is not accurate since this variable will hold more than just the data related to the decorative hero.